reg_sys/urls.py

```diff
@@ -0,0 +1,11 @@
+"""urls.py"""
```

Comment:
Use better documentation, what is the purpose of this file. You may refer to qr_web/urls.py